app/views/product_users/index.html.haml

```diff
@@ -20,6 +20,12 @@
   %p.notice= text("none")
 - else
   - if @product.has_product_access_groups?
+    = form_tag facility_instrument_users_path(format: :csv), method: :get, class: "search_form" do
```

Comment:
This needs to be pulled up to line 22, out of the conditional, so the link appears regardless of the presence of product access groups

Reply:
Oh I thought maybe we wouldn't want to display a link that would produce an empty file - but yeah I can see that it's good for consistency with how the rest of the site functions.

Reply:
If we have `@product_users` then we should have rows in the CSV export.  Putting the link on line 22 shouldn't create an empty file.  Is it possible you're mixing up `ProductAccessGroup` and `ProductUser`?